sdk/tests/conformance2/renderbuffers/framebuffer-test.html

```diff
@@ -188,6 +196,107 @@
   wtu.glErrorShouldBe(gl, gl.NO_ERROR,
             "detaching a renderbuffer from a framebuffer should succeed.");
 
+  // attach a 2d texture to read framebuffer, while no attachment to draw framebuffer.
+  gl.framebufferTexture2D(gl.READ_FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, fbtex, 0);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "attaching a texture to read framebuffer should succeed.");
+  gl.getFramebufferAttachmentParameter(gl.FRAMEBUFFER,
+                                       gl.COLOR_ATTACHMENT0,
+                                       desktopGL.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+  wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING on draw framebuffer without any attachment should generate INVALID_OPERATION.");
+  gl.getFramebufferAttachmentParameter(gl.READ_FRAMEBUFFER,
+                                       gl.COLOR_ATTACHMENT0,
+                                       // desktopGL.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+                                       gl.FRAMEBUFFER_ATTACHMENT_RED_SIZE);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING should generate INVALID_OPERATION.");
+
+  // attach different textures to read and draw framebuffer.
+  var fbtex1 = gl.createTexture();
+  gl.bindTexture(gl.TEXTURE_2D, fbtex1);
+  gl.texImage2D(gl.TEXTURE_2D, 0, gl.ALPHA, canvas.width, canvas.height, 0, gl.ALPHA, gl.UNSIGNED_BYTE, null);
```

Comment:
ALPHA type textures aren't required to be supported. See "Required Texture Formats" in section 3.8.3 "Texture Image Specification". I suggest you use something like an RG8 texture and check that the red channel has non-zero bits but the blue channel has zero bits instead.